lib/ngx/process.lua

```diff
@@ -32,6 +34,8 @@ int ngx_http_lua_ffi_enable_privileged_agent(char **err);
 int ngx_http_lua_ffi_get_process_type(void);
 void ngx_http_lua_ffi_process_signal_graceful_exit(void);
 int ngx_http_lua_ffi_master_pid(void);
+int ngx_http_lua_ffi_enable_sa_restart(int signum, int enabled);
+char *strerror(int errnum);
```

Comment:
imho, this _might_ belong to another `cdef` definition, if so please let me know where.

Reply:
This seems to break encapsulation. Better make the `ngx_http_lua_ffi_enable_sa_restart()` API function itself return the error string, just like our other C API functions?

Reply:
No problem, will change it :+1:

Reply:
Updated. Is it looking good enough for tests and documentation?